spec/src/main/asciidoc/microprofile-spec.asciidoc

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2017-2020 Contributors to the Eclipse Foundation
+// Copyright (c) 2017-2021 Contributors to the Eclipse Foundation
```

Comment:
do we need to/can we change the copyright if the file isn't changed in a different way?

Reply:
Good question, I'm not sure. Can anybody else confirm this? Otherwise, I will remove this change to be sure.

Reply:
Since the file has no changes, there is no need to update the end year.